bot/__main__.py

```diff
@@ -37,9 +39,29 @@
 bot = Bot(
     extensions,
     command_prefix=when_mentioned_or(config.COMMAND_PREFIX),
-    activity=Game(name=f"Ping me using {config.COMMAND_PREFIX}help"),
+    activity=Game(name=f"Invoke me using {config.COMMAND_PREFIX}help"),
     case_insensitive=True,
 )
 
+
+class TokenNotFoundError(Exception):
+    """
+    Raised if token is not found
+    """
+    pass
+
+
 if __name__ == "__main__":
-    bot.run(TOKEN)
+    if TOKEN is not None:
+        bot.run(TOKEN)
+    else:
+        colorama.init(autoreset=True)
+
+        raise TokenNotFoundError(
+            colorama.Fore.RED + "\n"
+            "Token is not found, are you sure\n"
+            "you are running this file through\n"
+            "pipenv and there is a .env file\n"
+            "containing a BOT_TOKEN key on your\n"
+            "current working directory?"
+        )
```

Comment:
```suggestion
        raise TokenNotFoundError(
            colorama.Fore.RED + "\n"
            "Token not found.\n"
            "Are you running this file through pipenv and\n"
            "is there a .env file containing a BOT_TOKEN key\n"
            "in your current working directory?\n"
        )
```
minor improvement to the error message